styles.css

```diff
@@ -0,0 +1,112 @@
+/* Adding poppins fonts */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  font-family: Poppins, sans-serif;
+}
+
+/* adding the main container */
+
+.container {
+  min-width: 375px;
+  max-width: 1200px;
+  padding-bottom: 70px;
+  background-color: #fff;
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 80px;
+  position: relative;
+  background-image: url(images/header-shapes-mobile.jpg);
```

Comment:
[Optional]

Kindly update the styling of your headline to match the figma design better🙏

Your design:
<img width="238" alt="image" src="https://github.com/ausiemass/portfolio-austin/assets/37116322/b5a5ef9a-862b-451a-83db-7d9c13471941">


Figma design:
<img width="216" alt="image" src="https://github.com/ausiemass/portfolio-austin/assets/37116322/746a5775-5c87-46f0-81a7-438fbb8fbdc2">

A few things to note:
- Please add the appropriate amountof padding on all sides in both the toolbar and the headline section
- Kindly align your social links to the left